### DISCUSSION TITLE
How do I initiate an index job?

### DISCUSSION THREAD

**Discussion by TheBig-O** [neonwatty/meme-search#50] [Q&A]
I'm running the pro version and can see the photos that need to be indexed, but nothing seems to work for starting a job. When I look in the `image_to_text_generator` container, it says `No jobs in queue. Waiting...`. 
After reading the README.MD, I tried to cd into the `meme_search/meme_search_pro/meme_search_app` folder, but there does not seem to be a folder that resembles ANY of those names.  
I have also tried running the command `rails test test/system` and that failes as well.  
If you could give us a few tips to get started, it would really help. I've been trying to get this running for several days now.  
Thank you!

**Comment by neonwatty** [comment]
Can you see any of your images under the `All memes` tab?  

At present due to the relatively large size of the present image-to-text model, automatic text generation for images is executed manually per image by clicking the "generate description" button on each image's card.   For most users it takes 1 - 2 minutes to generate text for an image automatically.  You can manually start generation for as many images as desired (that's where the queue comes in).

You can also manually enter this description per image.  

In either case - the text description of the image is indexed for keyword and vector search.

Perhaps its time for an optional "index all" images in a given subdirectory in the options screen - meaning that all have their text descriptions automatically generated and indexed.

**Reply by augmentedagent** [reply]
Any decisions on "index all" as a feature?  That would be perfect.

**Reply by neonwatty** [reply]
@augmentedagent thanks for the bump!

think i can get around to this in the next week!

**Reply by neonwatty** [reply]
@augmentedagent @TheBig-O - `index all` is now available as of version 2.0.0!  See example below:

![meme-search-bulk](https://github.com/user-attachments/assets/39fbd33b-1c06-4a89-a2ac-545b68a95f3a)

**Comment by neonwatty** [comment]
@augmentedagent @TheBig-O - `index all` is now available as of version 2.0.0!  See example below:

![meme-search-bulk](https://github.com/user-attachments/assets/39fbd33b-1c06-4a89-a2ac-545b68a95f3a)
